apps/src/templates/sectionsRefresh/coteacherSettings/AddCoteacher.jsx

```diff
@@ -37,7 +37,7 @@ const getErrorMessageFromResponse = (response, email) => {
       if (json.error.includes('inviting self')) {
         return i18n.coteacherCannotInviteSelf();
       }
-      if (json.error.includes('already in section')) {
+      if (json.error.includes('already a student')) {
         return i18n.coteacherAlreadyInCourse({email});
```

Comment:
maybe you can fix this in another PR if it's not breaking anything and you're in a hurry, but it looks like the `{email}` param you're passing here is unused

Reply:
Will add as a FLUP

Reply:
Fixed in https://github.com/code-dot-org/code-dot-org/pull/55053/files#diff-4fdbe7e710a3bca0b4c3e608efd9dc965091370a7361e13bc61d9cb6b8b530e6R41